Lib/email/mime/audio.py

```diff
@@ -31,14 +55,14 @@ def _whatsnd(data):
     """
     hdr = data[:512]
     fakefile = BytesIO(hdr)
-    for testfn in sndhdr.tests:
+    for testfn in _tests:
         res = testfn(hdr, fakefile)
```

Comment:
None of the inlined `test_*` functions use this second parameter `fakefile`, shall we remove it?

Reply:
I'm assuming @warsaw is going to rewrite all of this code anyway, so I will leave it to him. 😁

Reply:
Ha!  Yes, I will take a look when I get a chance.